README.rst

```diff
@@ -41,8 +41,8 @@ To install::
     $ python setup.py develop
 
 
-Setting up your project
-=======================
+Setting up your project by the defaults
```

Comment:
Still not 100 % clear to me, sorry. Maybe just stick to "Setting up your project" ...